workflow/snakemake_rules/main_workflow.smk

```diff
@@ -759,7 +761,7 @@ rule adjust_metadata_regions:
         Adjusting metadata for build '{wildcards.build_name}'
         """
     input:
-        metadata="results/{build_name}/metadata_with_index.tsv",
+        metadata="results/{build_name}/metadata_filtered.tsv.xz",
     output:
         metadata = "results/{build_name}/metadata_adjusted.tsv.xz"
```

Comment:
This changes the metadata that's input to several downstream rules. Rules which take `metadata_adjusted.tsv.xz` as input currently get the subsampled but _unfiltered_ metadata and with this change would get the subsampled and _filtered_ metadata.

Is this change a bug fix? It seems like it. But it does have potential visible changes like `rule colors` now will produce a scale based on the filtered rather than unfiltered metadata values.

I'd appreciate someone else's 👀 on this bit.

Reply:
/cc @nextstrain/core

Reply:
Split out separately in d8c17ec.